gitclone/git.go

```diff
@@ -115,46 +174,15 @@ func getDiffFile(buildURL, apiToken string, prID int) (string, error) {
 }
 
 func run(c *command.Model) error {
-	fmt.Println()
-	log.Infof("$ %s", c.PrintableCommandArgs())
-	var buffer bytes.Buffer
-
-	err := c.SetStdout(os.Stdout).SetStderr(io.MultiWriter(os.Stderr, &buffer)).Run()
-	if err != nil {
-		if errorutil.IsExitStatusError(err) {
-			return errors.New(strings.TrimSpace(buffer.String()))
-		}
-		return err
-	}
-
-	return nil
+	return runner.Run(c)
```

Comment:
What's the point of extracting this single line to a function?

Reply:
Similar thing in line 181 and 185.

Reply:
Removed extra indirection.